.github/workflows/test-dev.yml

```
@@ -20,14 +20,102 @@ permissions:
+        with:
+          imageName: ${{ env.CONTAINER_REGISTRY }}/${{ env.IMAGE_NAME }}
+          imageTag: ${{ env.IMAGE_TAG }}
+          configFile: .devcontainer/devcontainer-build.json
+          # push: always
+          push: never
+
   poetry-install:
     name: "Run: poetry install"
+    needs: devcontainer
+    env:
+      IMAGE_TAG: ${{ needs.devcontainer.output.IMAGE_TAG }}
```

actionlint (1.7.4)

property "output" is not defined in object type {outputs: {}; result: string}

(expression)